### DISCUSSION TITLE
supabase start - error

### DISCUSSION THREAD

**Discussion by SudarshanSaxena** [supabase/supabase#7477] [General]
installed supabase cli on docker container hosted on digitalocean
ran supabase start command and got this error

```
Error: Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/_ping": dial unix /var/run/docker.sock: connect: permission denied
  in github.com/supabase/cli/internal/utils.AssertDockerIsRunning:26
  in github.com/supabase/cli/internal/start.Run:35
```

**Comment by burggraf** [comment]
The container hosted on Digital Ocean may be out-of-date.  Maybe try setting it up manually using this: [https://supabase.com/docs/guides/hosting/overview](https://supabase.com/docs/guides/hosting/overview)

**Comment by kvetoslavnovak** [comment]
Same error here

**Reply by usaadi** [reply]
It should work if you add your user to the docker group and then reboot. (restarting docker service might not be enough)
